menuinst/platforms/win.py

```diff
@@ -271,14 +271,51 @@ def _command(self) -> str:
         if self.metadata["activate"]:
             conda_exe = self.menu.conda_exe
             if self.menu._is_micromamba(conda_exe):
-                activate = "shell activate"
+                activator = f'{self.menu.conda_exe} shell activate "{self.menu.prefix}"'
+                activation_lines = [
+                    f'@FOR /F "usebackq tokens=*" %%i IN (`{activator}`) do set "ACTIVATOR=%%i"',
+                    "@CALL %ACTIVATOR%",
+                ]
             else:
-                activate = "shell.cmd.exe activate"
-            activator = f'{self.menu.conda_exe} {activate} "{self.menu.prefix}"'
+                # conda >= 25.3.0 does not use .bat files to activate environments anymore.
+                # Instead, it produces .env files that are consumed inside
+                # conda/shell/condabin/_conda_activate.bat. There is no direct activator for this
+                # filetype, so menuinst has to parse the file and add the activator to the
+                # activation script directly.
+                activator_cmd = [
+                    str(self.menu.conda_exe),
+                    "shell.cmd.exe",
+                    "activate",
+                    str(self.menu.prefix),
+                ]
+                activator_run = logged_run(activator_cmd, check=True, log=False)
+                activation_file = Path(activator_run.stdout.strip())
+                filetype = activation_file.suffix
+                if filetype == ".bat":
+                    activator = (
+                        f'{self.menu.conda_exe} shell.cmd.exe activate "{self.menu.prefix}"'
```

Comment:
```suggestion
                        f'"{self.menu.conda_exe}" shell.cmd.exe activate "{self.menu.prefix}"'
```

Is this quoted at source? If not, we should quote it.

Reply:
It is currently not quoted and I didn't find a difference in behavior whether we quote it or not. This does bring me to a different question though: we may need a separate test branch for `conda < 25.3.x` for Windows if we want to test that branch since the latest conda version will be a moving target. Thoughts?

Reply:
We can add a new CI job with these constraints while we still support that version, `menunist` is pretty cheap CI-wise thankfully.

Reply:
Turns out that quoting is a problem on some systems. https://github.com/conda/menuinst/pull/330/commits/6de834d12fd0b9c14f5a7d7f57e998ffdc203a64 got the tests that used `bat` files working again.